tycho-extras/tycho-eclipserun-plugin/src/main/java/org/eclipse/tycho/extras/eclipserun/EclipseRunMojo.java

```diff
@@ -172,8 +171,8 @@ public class EclipseRunMojo extends AbstractMojo {
 	 * 
 	 * <pre>
 	 * &lt;applicationsArgs&gt;
-	 *   &lt;args&gt;-buildfile&lt;/args&gt;
-	 *   &lt;args&gt;build-test.xml&lt;/args&gt;
+	 *   &lt;arg&gt;-buildfile&lt;/arg&gt;
```

Comment:
Same comment about javadoc readability in the code.

Reply:
Done